src/bun.js/test/jest.zig

```diff
@@ -1459,20 +1459,33 @@ pub const TestRunnerTask = struct {
     };
 
     pub fn handleResult(this: *TestRunnerTask, result: Result, from: ResultType) void {
+        var result_copy = result;
+        this.handleResultPtr(&result_copy, from);
```

Comment:
Is it important some calls to `handleResult` do not modify `result`? If not, I think `handleResultPtr` should be deleted and `handleResult` should take a pointer